packages/apollo/src/Command.ts

```diff
@@ -128,6 +139,19 @@ export abstract class ProjectCommand extends Command {
       });
     }
 
+    if (flags.localSchemaFile) {
```

Comment:
I agree that it may work but isn't ideal. This is better!

```javascript
if (flags.localSchemaFile) {
      if (isClientConfig(config)) {
        config.setDefaults({
          client: {
            service: {
              localSchemaFile: flags.localSchemaFile
            }
          }
        });
      } else if (isServiceConfig(config)) {
        config.setDefaults({
          service: {
            localSchemaFile: flags.localSchemaFile
          }
        });
      }
```